programs/0000-template.md

```diff
@@ -0,0 +1,41 @@
+- **Start Date:** (fill me in with today's date, YYYY-MM-DD)
```

Comment:
this should likely be removed and punted to https://github.com/nodejs/bootstrap/pull/23